packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-reset-password/amplify-confirm-reset-password.component.ts

```diff
@@ -1,32 +1,32 @@
-import { Component, HostBinding, Input, OnInit } from '@angular/core';
+import { Component, HostBinding, Input } from '@angular/core';
 import { AuthenticatorService } from '../../../../services/authenticator.service';
 import {
   FormFieldsArray,
   getFormDataFromEvent,
-  hasTranslation,
-  translate,
+  authenticatorTextUtil,
 } from '@aws-amplify/ui';
 
+const {
+  getBackToSignInText,
+  getResendCodeText,
+  getSubmitText,
+  getResetYourPasswordText,
+} = authenticatorTextUtil;
```

Comment:
Rest of the Angular Authenticator changes are using this `authenticatorTextUtil` in favor of hard coded string.